portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,92 @@
+html{
+    background-color: rgb(17, 15, 15);
+    color: #fcf4f4;
+}
+
+body{
+    max-width: 800px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#navbar{
+    grid-area: header;
+    font-size: 20px;
+}
+
+#navbar .navbar-item{
+    padding:8px 16px;
+    float:left;
+    width:auto;
+    border:none;
+    display:block;
+    outline:0s;
+}
+
+.nav-button{
+    border:none;
+    display:inline-block;
+    padding:8px 16px;
+    vertical-align:middle;
+    overflow:hidden;
+    text-decoration:none;
+    color:inherit;
+    background-color:inherit;
+    text-align:center;
+    cursor:pointer;
+    white-space:nowrap
+}
+
+
+#nav-left-side{
+    float: left;
+}
+
+#nav-right-side{
+    float: right;
+}
+
+.clear{
+    clear: both;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-top: 30%;
+    display: grid;
+    margin-left: auto;
+    margin-right: auto;
+    grid-template-columns: 550px 200px auto;
+    column-gap: 0px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+
+#content-body{
+    align-self: center;
+    position: relative;
+    padding: 10px 25px;
+    left: 20px;
+    z-index: -1;
 }
+
+#content-body > h1{
+    font-size: 32px;
+}
+
+#content-body > p{
+    font-size: 16px;
+}
+
+#content-body > #main-content-body{
+    font-size: 22px;
+}
+
+#content-image{
+    align-self: center;
+    z-index: 100;
+}
+
+#content-image img{
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+}
```

Comment:
It's recommended to have an empty line at the bottom of every file (it's usually for file utils that read in a file and expect a new line at the end, and usually gets auto appended by text editors, but good habit to pick up as well :))